Makefile

```diff
@@ -93,8 +100,16 @@ python-init-dev-only:
 python-init-test-only: lib/test-requirements.txt
 	INSTALL_DEV_REQS=false INSTALL_TEST_REQS=true make python-init
 
+.PHONY: python-version-check
+# Check if the Python version exists.
+python-version-check:
+	@if [ -z "$(PYTHON_VERSION)" ]; then \
+        echo "Error: Python version is not detected. Please ensure Python is installed and accessible in your PATH."; \
+        exit 1; \
+    fi
+
 .PHONY: python-init
-python-init:
+python-init: python-version-check
```

Comment:
I think these additions can now be removed in favor of https://github.com/streamlit/streamlit/pull/9930/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R31-R35

Reply:
Removed.